tools/github-event-processor/README.md

```diff
@@ -0,0 +1,108 @@
+# GitHub Event Processor
+
+## Overview
+
+GitHub Event Processor is an Azure-SDK replacement for FabricBot. It's written in C# using [Octokit.Net](https://github.com/octokit/octokit.net). Where FabricBot is a separate service, GitHub Event Processor will utilize Action and Scheduled events, triggered through [GitHub Action Workflows](https://docs.github.com/en/actions/using-workflows/about-workflows). These are defined YML files and placed into the .github/workflows directory of the repository utilizing them. For our purposes there will be two YML files, one for Actions and one for Scheduled events.
```

Comment:
I don't think we need to really talk about Fabricbot in our persisted documentation.

Reply:
Will be removed with the next commit.